lib/spack/spack/build_systems/cuda.py

```diff
@@ -93,12 +93,16 @@ def cuda_flags(arch_list):
     conflicts('%clang@7:', when='+cuda ^cuda@10.0.130' + arch_platform)
 
     # Intel is mostly relevant for x86_64 Linux, even though it also
-    # exists for Mac OS X.
-    conflicts('%intel@:14,16:', when='+cuda ^cuda@7.5')
-    conflicts('%intel@:14,17:', when='+cuda ^cuda@8.0.44')
-    conflicts('%intel@:14,18:', when='+cuda ^cuda@8.0.61:9.1')
-    conflicts('%intel@17:18', when='+cuda ^cuda@9.2:')
-    conflicts('%intel@19:', when='+cuda')
+    # exists for Mac OS X. No information prior to CUDA 3.2 or Intel 11.1
+    conflicts('%intel@:11.0', when='+cuda ^cuda@:3.1')
+    conflicts('%intel@:12.0', when='+cuda ^cuda@5.5:')
+    conflicts('%intel@:13.0', when='+cuda ^cuda@6.0:')
```

Comment:
according to my [recordings](https://gist.github.com/ax3l/9489132), `intel@13.1` is the only supported version for CUDA 6.0